javascript/javascript1/week2/FlightBooking.js

```diff
@@ -0,0 +1,19 @@
+function getFullname(firstname,surname) {
+    return firstname,surname;
+}
+console.log("saloumeh","sarabi");
+
+let fullname1 = getFullname; 
+let fullname2 = getFullname;
+console.log("saloumeh sarabi,farzad soleimani");
+
+var gender = prompt('are you lord or princess')
+if (gender === 'lord') {
+  console.log(getFullname(' lord',' farzad', " soleimani ",))
+} else {
+  console.log(getFullname('princess', ' saloumeh', " sarabi "))
+}
```

Comment:
Have you checked on your console? It actually is not working and is not supposed to be. Your function has 2 parameters while you are calling the function with 3 parameters. Find out a way to make it work :)